.circleci/config.yml

```diff
@@ -1,181 +1,71 @@
-          command: ./gradlew publish
-      - run:
-          name: Publish Documentation
-          command: |
-            set -e
-            ./gradlew generatePublishableDocs
-            if [[ ! $CIRCLE_TAG == *"-"* ]]; then
-              git checkout gh-pages
-              cp -R miniapp/build/publishableDocs/docs/. ./docs
-              cp -R miniapp/build/publishableDocs/_versions/. ./_versions
-              git add docs _versions
-              git config user.name "CI Publisher"
-              git config user.email "dev-opensource@mail.rakuten.com"
-              git commit -m "Publish documentation for $CIRCLE_TAG"
-              git push origin gh-pages
-            else
-              echo "Documentation not published for snapshot version"
-            fi
-
-  deploy-test-app-stg:
-    docker:
-      - image: circleci/node
-    working_directory: ~/code
-    steps:
-      - attach_workspace:
-          at: ./
-      - run:
-          name: Install App Center CLI
-          command: npm init --yes && npm install appcenter-cli
-      - run:
-          name: Upload APK to App Center
-          command: >
-            npx appcenter distribute release
-            --group Testers
-            --file apk/staging/testapp-staging.apk
-            --app $APP_CENTER_APP_NAME
-            --release-notes $CIRCLE_BUILD_URL
-            --token $APP_CENTER_TOKEN
-            --quiet
-
-  deploy-test-app-prod:
-    docker:
-      - image: circleci/node
-    working_directory: ~/code
-    steps:
-      - attach_workspace:
-          at: ./
-      - run:
-          name: Install App Center CLI
-          command: npm init --yes && npm install appcenter-cli
-      - run:
-          name: Upload APK to App Center
-          command: >
-            npx appcenter distribute release
-            --group Production
-            --file apk/release/testapp-release.apk
-            --app $APP_CENTER_APP_NAME
-            --release-notes "Production build for Android SDK $CIRCLE_TAG"
-            --token $APP_CENTER_TOKEN
-            --quiet
+orbs:
+  android-sdk: rakutentech/android-sdk@0.1.0
+  app-center: rakutentech/app-center@0.1.0
 
 workflows:
-  version: 2
+  version: 2.1
   build-and-release:
     jobs:
-      - build:
+      - android-sdk/build:
+          gradle-cache-key: >-
+            gradle-
+            {{ checksum "build.gradle" }}
+            {{ checksum "miniapp/build.gradle" }}
+            {{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+          maven-cache-key: maven-{{ checksum "miniapp/src/test/AndroidManifest.xml" }}
+          sdk-path: miniapp
+          sample-app-path: testapp
+          pre-steps:
+            # Retrieve Base64 Keystore from Env variable and save to testapp project
+            - run: |
+                if [[ "$RELEASE_KEYSTORE_BASE64" != "" ]]; then
+                  base64 -d \<<< "$RELEASE_KEYSTORE_BASE64" > ./testapp/release-keystore.jks
+                fi
```

Comment:
out of curiosity: this keystore is only for assembling a release variant of the test app, right? 
If so - why do we treat it as a secret? (my assumption is that injecting the base64 encoded keystore is to keep it secret from the source code)

Reply:
Yeah, it is just for the test app so it's not really critical that it's kept secret. But it seems better to be on the safe side and keep it secret if possible. We distribute the release variant to external customers to use for testing Mini Apps.